app/services/permissions_checker.rb

```diff
@@ -56,4 +58,14 @@ def authorize_manage_recordings
 
     @current_user.recordings.find_by(record_id: @record_id).present?
   end
+
+  def authorize_room_limit
+    user = User.find(@user_id)
```

Comment:
We may like to be consistent with the rest of the `authorize_*` methods by returning `false` directly if the `@user_id` is `blank?`.